strategy/api/resources/templates/class.cfm

```diff
@@ -688,6 +688,28 @@ Class
 		}
 		*/
 
+		/**
+		 * Improved version of Ben Forta's "ParagraphFormat2" method on cflib.
```

Comment:
What's the difference in just using `paragraphFormat`? https://cfdocs.org/paragraphformat